railties/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+*   Allow configuration of eager_load behaviour for rake environment:
```

Comment:
I would say "class eager_load` behavior" here to avoid potential confusions with ActiveRecord eager loading. I get that we're not in the AR changelog but you never know. :-p